HADOOP-17678. Add Dockerfile for Centos 7

GauthamBanasandra opened the pull request:
* Adding a Dockerfile for building on Centos 7
  since some folks in the Hadoop community
  are using Centos 7.

GauthamBanasandra committed:
Add Dockerfile for Centos 7

* Creating image for building
  on Centos  7.

GauthamBanasandra committed:
Do full build on Centos 7

* Built Hadoop on Centos 7.

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/1/console in case of problems.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 35s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  1s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  1s |  |  codespell was not available.  |
| +0 :ok: |  shelldocs  |   0m  1s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |  16m 14s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  shadedclient  |  13m 17s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 33s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| -1 :x: |  hadolint  |   0m  2s | [/results-hadolint.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/1/artifact/out/results-hadolint.txt) |  The patch generated 5 new + 0 unchanged - 0 fixed = 5 total (was 0)  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  |  No new issues.  |
| +1 :green_heart: |  shadedclient  |  13m  6s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 36s |  |  The patch does not generate ASF License warnings.  |
|  |   |  46m 16s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2967 |
| Optional Tests | dupname asflicense codespell hadolint shellcheck shelldocs |
| uname | Linux 1e512aafdc2e 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / fac7600eaf1cd14c32bb6176eda610c0254f0da8 |
| Max. process+thread count | 666 (vs. ulimit of 5500) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/1/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.0 hadolint=1.11.1-0-g0e692dd |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Fix warnings

* Fixed the warnings
  reported in the CI
  run.

GauthamBanasandra committed:
Ignore hadolint warnings

* Ignoring some hadolint
  warnings.

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/2/console in case of problems.

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/3/console in case of problems.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 37s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |  16m 29s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  shadedclient  |  13m 14s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 32s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| -1 :x: |  hadolint  |   0m  1s | [/results-hadolint.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/2/artifact/out/results-hadolint.txt) |  The patch generated 1 new + 0 unchanged - 0 fixed = 1 total (was 0)  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  |  No new issues.  |
| +1 :green_heart: |  shadedclient  |  13m  8s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 37s |  |  The patch does not generate ASF License warnings.  |
|  |   |  46m 33s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2967 |
| Optional Tests | dupname asflicense codespell hadolint shellcheck shelldocs |
| uname | Linux 9f996e3a06a3 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 14102b6edc858a02322fbdf9145fdd6f06ea2432 |
| Max. process+thread count | 663 (vs. ulimit of 5500) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/2/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.0 hadolint=1.11.1-0-g0e692dd |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 51s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |  16m  9s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  shadedclient  |  13m 17s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 31s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| -1 :x: |  hadolint  |   0m  1s | [/results-hadolint.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/3/artifact/out/results-hadolint.txt) |  The patch generated 1 new + 0 unchanged - 0 fixed = 1 total (was 0)  |
| +1 :green_heart: |  shellcheck  |   0m  1s |  |  No new issues.  |
| +1 :green_heart: |  shadedclient  |  12m 49s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 36s |  |  The patch does not generate ASF License warnings.  |
|  |   |  46m  5s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2967 |
| Optional Tests | dupname asflicense codespell hadolint shellcheck shelldocs |
| uname | Linux cd309ef3e6ff 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 14102b6edc858a02322fbdf9145fdd6f06ea2432 |
| Max. process+thread count | 717 (vs. ulimit of 5500) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/3/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.0 hadolint=1.11.1-0-g0e692dd |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Fix warning

* Fixed warning reported
  by the CI run.

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/4/console in case of problems.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 35s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |  16m  8s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  shadedclient  |  13m 25s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 32s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| -1 :x: |  hadolint  |   0m  1s | [/results-hadolint.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/4/artifact/out/results-hadolint.txt) |  The patch generated 2 new + 0 unchanged - 0 fixed = 2 total (was 0)  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  |  No new issues.  |
| +1 :green_heart: |  shadedclient  |  12m 58s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 38s |  |  The patch does not generate ASF License warnings.  |
|  |   |  46m 10s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/4/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2967 |
| Optional Tests | dupname asflicense codespell hadolint shellcheck shelldocs |
| uname | Linux ae4d1f8fa9ca 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / d186cd1ff22ac648b787f7fc11b34986d3d47d7e |
| Max. process+thread count | 694 (vs. ulimit of 5500) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/4/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.0 hadolint=1.11.1-0-g0e692dd |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Do not build Hadoop

* Removed code for
  building Hadoop.

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/5/console in case of problems.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 35s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |  16m 18s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  shadedclient  |  13m 25s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 33s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| -1 :x: |  hadolint  |   0m  2s | [/results-hadolint.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/5/artifact/out/results-hadolint.txt) |  The patch generated 1 new + 0 unchanged - 0 fixed = 1 total (was 0)  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  |  No new issues.  |
| +1 :green_heart: |  shadedclient  |  13m  8s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 38s |  |  The patch does not generate ASF License warnings.  |
|  |   |  46m 31s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/5/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2967 |
| Optional Tests | dupname asflicense codespell hadolint shellcheck shelldocs |
| uname | Linux ad9c75059422 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 0445608730342fee861692ebd03b04af52ae6700 |
| Max. process+thread count | 669 (vs. ulimit of 5500) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/5/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.0 hadolint=1.11.1-0-g0e692dd |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Fix warning SC2164

* Fixed warning SC2164
  reported in CI run by
  exiting when cd fails.

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/6/console in case of problems.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 33s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  1s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  1s |  |  codespell was not available.  |
| +0 :ok: |  shelldocs  |   0m  1s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |  16m 11s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  shadedclient  |  13m 34s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 32s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| -1 :x: |  hadolint  |   0m  2s | [/results-hadolint.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/6/artifact/out/results-hadolint.txt) |  The patch generated 1 new + 0 unchanged - 0 fixed = 1 total (was 0)  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  |  No new issues.  |
| +1 :green_heart: |  shadedclient  |  13m  0s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 38s |  |  The patch does not generate ASF License warnings.  |
|  |   |  46m 21s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/6/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2967 |
| Optional Tests | dupname asflicense codespell hadolint shellcheck shelldocs |
| uname | Linux bcf4ac1e3484 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / f277e5971425863bd94e4a2dab5dee90886640a7 |
| Max. process+thread count | 742 (vs. ulimit of 5500) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/6/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.0 hadolint=1.11.1-0-g0e692dd |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Add exit for cd

* The script to exit
   if cd fails for any
   reason.

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/7/console in case of problems.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 49s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |  16m  4s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  shadedclient  |  13m 26s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 32s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| -1 :x: |  hadolint  |   0m  2s | [/results-hadolint.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/7/artifact/out/results-hadolint.txt) |  The patch generated 1 new + 0 unchanged - 0 fixed = 1 total (was 0)  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  |  No new issues.  |
| +1 :green_heart: |  shadedclient  |  12m 57s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 35s |  |  The patch does not generate ASF License warnings.  |
|  |   |  46m 15s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/7/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2967 |
| Optional Tests | dupname asflicense codespell hadolint shellcheck shelldocs |
| uname | Linux b7694d6fcfa1 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 513f23cfe3765428902b40f228e13b1529c490c0 |
| Max. process+thread count | 745 (vs. ulimit of 5500) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/7/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.0 hadolint=1.11.1-0-g0e692dd |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Add exit for cd

* Had missed adding
   an exit for last cd.

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/8/console in case of problems.

hadoop-yetus commented:
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 59s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |  16m  6s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  shadedclient  |  14m  2s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 29s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  hadolint  |   0m  2s |  |  No new issues.  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  |  No new issues.  |
| +1 :green_heart: |  shadedclient  |  13m 14s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 36s |  |  The patch does not generate ASF License warnings.  |
|  |   |  47m 19s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/8/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2967 |
| Optional Tests | dupname asflicense codespell hadolint shellcheck shelldocs |
| uname | Linux 06bcc82fced0 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / d93ffe6cb3cd157d094d2b62f0618ff4f37969be |
| Max. process+thread count | 719 (vs. ulimit of 5500) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/8/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.0 hadolint=1.11.1-0-g0e692dd |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra commented:
@aajisaka , @iwasakims , @ericbadger I've created Dockerfile for Centos 7. I'll be creating similar Dockerfiles for the platforms mentioned in https://issues.apache.org/jira/browse/BIGTOP-3471. Anyone can use this to check for compilation failures for the PRs.

GauthamBanasandra committed:
Remove redundant packages

* Removed some packages
  that aren't used by build.

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/9/console in case of problems.

hadoop-yetus commented:
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 54s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |  16m  4s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  shadedclient  |  13m 49s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 31s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  hadolint  |   0m  1s |  |  No new issues.  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  |  No new issues.  |
| +1 :green_heart: |  shadedclient  |  13m 21s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 34s |  |  The patch does not generate ASF License warnings.  |
|  |   |  47m 10s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/9/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2967 |
| Optional Tests | dupname asflicense codespell hadolint shellcheck shelldocs |
| uname | Linux d1fad05a45a4 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 3d4e3963d214c16521232bc1b5fe55eae40b2c01 |
| Max. process+thread count | 542 (vs. ulimit of 5500) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/9/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.0 hadolint=1.11.1-0-g0e692dd |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m  0s |  |  Docker mode activated.  |
| -1 :x: |  docker  |   0m 19s |  |  Dockerfile 'src/dev-support/docker/Dockerfile_centos_7' not found.  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| GITHUB PR | https://github.com/apache/hadoop/pull/2967 |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/10/console |
| versions | git=2.17.1 |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/11/console in case of problems.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 51s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  34m 23s |  |  trunk passed  |
| +1 :green_heart: |  mvnsite  |  25m 44s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  13m 28s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  19m 40s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  1s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  hadolint  |   0m  2s |  |  No new issues.  |
| +1 :green_heart: |  mvnsite  |  20m  8s |  |  the patch passed  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  |  No new issues.  |
| +1 :green_heart: |  shadedclient  |  13m 25s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  unit  |  19m 20s |  |  root in the patch passed.  |
| +1 :green_heart: |  asflicense  |   1m  6s |  |  The patch does not generate ASF License warnings.  |
|  |   | 150m 21s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/11/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2967 |
| Optional Tests | dupname asflicense codespell shellcheck shelldocs hadolint mvnsite unit |
| uname | Linux cc3204c666ee 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 26ae3b7acdaa9b68615f94aee71a7124b6c995a7 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/11/testReport/ |
| Max. process+thread count | 542 (vs. ulimit of 5500) |
| modules | C: . U: . |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/11/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.0 hadolint=1.11.1-0-g0e692dd |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/12/console in case of problems.

hadoop-yetus commented:
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 55s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |  16m 14s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  shadedclient  |  14m 27s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 31s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  hadolint  |   0m  2s |  |  No new issues.  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  |  No new issues.  |
| +1 :green_heart: |  shadedclient  |  13m  3s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 36s |  |  The patch does not generate ASF License warnings.  |
|  |   |  47m 41s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/12/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2967 |
| Optional Tests | dupname asflicense codespell hadolint shellcheck shelldocs |
| uname | Linux ca91f5dd05ed 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 3d4e3963d214c16521232bc1b5fe55eae40b2c01 |
| Max. process+thread count | 629 (vs. ulimit of 5500) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/12/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.0 hadolint=1.11.1-0-g0e692dd |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Install dependencies using yum

* Installed some dependencies
  using yum.

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/13/console in case of problems.

hadoop-yetus commented:
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   1m 26s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |  16m 33s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  shadedclient  |  13m 43s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 32s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  hadolint  |   0m  2s |  |  No new issues.  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  |  No new issues.  |
| +1 :green_heart: |  shadedclient  |  13m 16s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 35s |  |  The patch does not generate ASF License warnings.  |
|  |   |  48m  4s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/13/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2967 |
| Optional Tests | dupname asflicense codespell hadolint shellcheck shelldocs |
| uname | Linux f90570be6022 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 7660c82b003d63fbbedb66a5a3f562c22b8bd8fe |
| Max. process+thread count | 742 (vs. ulimit of 5500) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/13/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.0 hadolint=1.11.1-0-g0e692dd |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Install node.js

* Installing latest node.js
  separately.

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/14/console in case of problems.

hadoop-yetus commented:
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 52s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |  17m 47s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  shadedclient  |  13m 28s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 31s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| -1 :x: |  hadolint  |   0m  1s | [/results-hadolint.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/14/artifact/out/results-hadolint.txt) |  The patch generated 1 new + 0 unchanged - 0 fixed = 1 total (was 0)  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  |  No new issues.  |
| +1 :green_heart: |  shadedclient  |  13m  4s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 36s |  |  The patch does not generate ASF License warnings.  |
|  |   |  48m 10s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/14/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2967 |
| Optional Tests | dupname asflicense codespell hadolint shellcheck shelldocs |
| uname | Linux 29134aa5a53b 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / b6bab6e6adc9936f3951dc1c0173db2e4f91710b |
| Max. process+thread count | 717 (vs. ulimit of 5500) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/14/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.0 hadolint=1.11.1-0-g0e692dd |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Fix SC2164

* Fixed SC2164 warning
  raised during CI run.

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/15/console in case of problems.

hadoop-yetus commented:
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 49s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  shelldocs  |   0m  0s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |  16m 46s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  shadedclient  |  13m 13s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 32s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  hadolint  |   0m  1s |  |  No new issues.  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  |  No new issues.  |
| +1 :green_heart: |  shadedclient  |  13m  9s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 36s |  |  The patch does not generate ASF License warnings.  |
|  |   |  47m  2s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/15/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2967 |
| Optional Tests | dupname asflicense codespell hadolint shellcheck shelldocs |
| uname | Linux 28ceda18115f 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 904326ec16dda742dc4e5ed22998690e4ad85701 |
| Max. process+thread count | 752 (vs. ulimit of 5500) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/15/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.0 hadolint=1.11.1-0-g0e692dd |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

GauthamBanasandra committed:
Install bower dependency

* Had commented out
  installation of bower
  since there were issues
  in installing node.js.
* We're now installing
  bower since node.js
  is now installed
  separately.

hadoop-yetus commented:
(!) A patch to the testing environment has been detected. 
Re-executing against the patched versions to perform further tests. 
The console is at https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/16/console in case of problems.

hadoop-yetus commented:
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 52s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  0s |  |  codespell was not available.  |
| +0 :ok: |  shelldocs  |   0m  1s |  |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
|||| _ trunk Compile Tests _ |
| +0 :ok: |  mvndep  |  16m 19s |  |  Maven dependency ordering for branch  |
| +1 :green_heart: |  shadedclient  |  13m 21s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 32s |  |  Maven dependency ordering for patch  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  hadolint  |   0m  2s |  |  No new issues.  |
| +1 :green_heart: |  shellcheck  |   0m  0s |  |  No new issues.  |
| +1 :green_heart: |  shadedclient  |  13m  3s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 35s |  |  The patch does not generate ASF License warnings.  |
|  |   |  46m 32s |  |  |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/16/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/2967 |
| Optional Tests | dupname asflicense codespell hadolint shellcheck shelldocs |
| uname | Linux b10a38aa982d 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 8d42a0963a6252147eb68cecf1a4d7436d3e5da6 |
| Max. process+thread count | 628 (vs. ulimit of 5500) |
| modules | C:  U:  |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-2967/16/console |
| versions | git=2.25.1 maven=3.6.3 shellcheck=0.7.0 hadolint=1.11.1-0-g0e692dd |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.
